packages/holocron/__tests__/loadModule.node.spec.js

```diff
@@ -62,13 +57,26 @@ describe('loadModule.node', () => {
     return require('../src/loadModule.node.js').default; // eslint-disable-line global-require
   }
 
+  function getSHA(content = 'the Spanish Inquisition') {
+    // eslint-disable-next-line global-require
+    return require('ssri').fromData(content, { algorithms: ['sha256', 'sha384'] }).toString();
+  }
+
+  beforeAll(() => {
+    jest.spyOn(console, 'log');
+    jest.spyOn(console, 'warn');
+    console.log.mockImplementation(() => { });
+    console.warn.mockImplementation(() => { });
+  });
+
   beforeEach(() => {
-    console.log.mockClear();
-    console.log.mockImplementation(() => {});
-    console.warn.mockClear();
-    console.warn.mockImplementation(() => {});
+    jest.clearAllMocks();
+  });
+
+  const { NODE_ENV } = process.env;
 
-    fetch.mockClear();
+  afterEach(() => {
+    process.env.NODE_ENV = NODE_ENV;
```

Comment:
the test suite is run with `NODE_ENV=development jest` which needs to be reset for tests to pass.